conda/environments/cuspatial_dev_cuda10.0.yml

```diff
@@ -9,3 +9,4 @@ dependencies:
   - cudf=0.14.*
   - cudatoolkit=10.0
   - gdal>=2.4.*
+  - geopandas=0.7.0
```

Comment:
Is this dependency just for the notebook?

Also, it needs to be added to every environment file.

Reply:
I'll add it to each env.  I think it is for this and any other notebook that uses geopandas.  This was an @thomcom add, so i trust its necessity, https://github.com/taureandyernv/cuspatial/commit/4e2c3ecfb1f9b83152f4acecf8200ba3fb218b2e#diff-5a99ec3304f1f4197b04cffc0fa980c1.

Reply:
@raydouglass i think this was resolved.  can you check the changes?

Reply:
Thanks for looking into this. I'm planning on making `geopandas` a regular dependency for the python portion of cuSpatial, and it made building this notebook easier.